test/common-chunks/webpack.config.js

```diff
@@ -14,14 +26,7 @@ module.exports = {
         filename: '[name].[chunkhash].js',
     },
     plugins: [
-        new OutputHash({
-            manifestFiles: ['vendor'],
```

Comment:
We are not testing this functionality anymore?

Reply:
This is tested as part of the manifest tests instead of the common chunks.